dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -254,6 +254,39 @@ public int ResultsPerPrompt
         }
     }
 
+    /// <summary>
+    /// Whether to return log probabilities of the output tokens or not.
+    /// If true, returns the log probabilities of each output token returned in the `content` of `message`.
+    /// </summary>
+    [Experimental("SKEXP0010")]
+    [JsonPropertyName("logprobs")]
+    public bool? Logprobs
+    {
+        get => this._logprobs;
+
+        set
+        {
+            this.ThrowIfFrozen();
+            this._logprobs = value;
+        }
+    }
+
+    /// <summary>
+    /// An integer specifying the number of most likely tokens to return at each token position, each with an associated log probability.
+    /// </summary>
+    [Experimental("SKEXP0010")]
```

Comment:
I don't think we need to make this experimental as this is just a wrapper around what is in the OpenAI API.

Is there any uncertainty here?

Reply:
> I don't think we need to make this experimental as this is just a wrapper around what is in the OpenAI API.

@markwallace-microsoft I followed the same approach we used when added `Seed` field in this PR:
https://github.com/microsoft/semantic-kernel/pull/4162/files#diff-fffa27d3df6f16d9f8a41044ade5f40246999ce8de832769b7ca91f8b94891d4R73
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/cf34c714-25d5-4785-86d6-77c30645e6ea)

I also think that even if it's just a wrapper, it's better to mark it experimental, keep it for a couple of weeks and in case there are no issues - remove `Experimental` attribute. It gives us some space for bug fixing if any.